### ISSUE TITLE
Maximum call stack size exceeded when using sent_payload_callback

### ISSUE THREAD

**Issue by vesper8** [permafrost-dev/vue-ray#11]
I tried to add your example as-is just to test. I added this to my `vue-ray` config

```
  sent_payload_callback: (rayInstance) => {
    // automatically make all payloads sent to Ray green.
    rayInstance.green();
  },
```

Then, in a view I added

```
  mounted() {
    console.log('console.log test');
    this.$ray('testing');
  },
```

It correctly styled the `console.log` with the green colour, but then it froze the page and eventually spat out the error below:

![Screen Shot 2021-04-26 at 10 59 34 PM](https://user-images.githubusercontent.com/816028/116178758-d76c7980-a71e-11eb-8c23-6be829fc8ee7.png)

**Comment by patinthehat** [comment]
@vesper8 Thank you for the bug report, I'm looking into this now. :+1:

**Comment by vesper8** [comment]
thank you! : )

**Comment by vesper8** [comment]
Any luck finding what is causing this issue? It's still happening as of `1.17.2`

**Comment by patinthehat** [comment]
@vesper8 This issue has been resolved with `node-ray` v1.14.0.  Updating your deps should resolve the issue.
Thank you!
